src/racer/matchers.rs

```diff
@@ -298,7 +300,7 @@ pub fn match_mod(msrc: Src, blobstart: usize, blobend: usize,
                 contextstr: filepath.to_str().unwrap().to_owned(),
                 generic_args: Vec::new(),
                 generic_types: Vec::new(),
-                docs: String::new(),  //TODO: fix
+                docs: String::new(),
```

Comment:
I was thinking that `String::new` allocated, but that's simply not true. Thanks for making me look it up! The optionless version is a much nicer API.